operations.md

```diff
@@ -25,6 +26,22 @@ curl -X GET "http://localhost:11211/admin/log"
 
 Every error message is identified with a prefix code in brackets. The code convention can be found in Apendix A.
 
+## ## <a name="Cache Stats"/> Cache Stats
+
+PEP keeps a memory cache with some access about roles, domains, users and subservices. Related with this info is possible
+get an statistics about these caches using the following API:
+
+```
+curl -X GET "http://localhost:11211/admin/cacheStats"
+```
+Getting
+
+`json
+{"cacheStats":{"subservice":{"hits":0,"misses":0,"keys":0,"ksize":0,"vsize":0},"roles":{"hits":9,"misses":1,"keys":0,"ksize":0,"vsize":0},"user":{"hits":9,"misses":2,"keys":1,"ksize":183,"vsize":240},"validation":{"hits":9,"misses":1,"keys":0,"ksize":0,"vsize":0}}}
```

Comment:
Rendering look a bit ugly...

![imagen](https://user-images.githubusercontent.com/1534240/169067722-0bec0b59-fded-4f28-aac7-537d92b93e4d.png)

I'd suggest to use ` ```json` (triple quote) and pretty print

Reply:
done in https://github.com/telefonicaid/fiware-pep-steelskin/pull/509/commits/d2955757ebc45515e0344e0868640d703a4799ad